packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -11,19 +11,29 @@ interface ComboboxContentProps {
   readonly options: string[];
 
   /**
-   * onSelection handler.
+   * onSelect handler.
```

Comment:
Hmm... we should explain this better for the newer peeps who aren't familiar with the system. Also stating intent.

> Callback that fires whenever an option is selected.

And probably something along the lines of the selected params is the label of the option?